src/src/DataStructs/ExtraTaskSettingsStruct.h

```diff
@@ -82,6 +82,7 @@ struct ExtraTaskSettingsStruct
   char        TaskDeviceName[NAME_FORMULA_LENGTH_MAX + 1];
   char        TaskDeviceFormula[VARS_PER_TASK][NAME_FORMULA_LENGTH_MAX + 1];
   char        TaskDeviceValueNames[VARS_PER_TASK][NAME_FORMULA_LENGTH_MAX + 1];
+  char        TaskDeviceEArgs[VARS_PER_TASK][NAME_FORMULA_LENGTH_MAX + 1];
```

Comment:
This struct is stored as is in the settings file.
If you like to add something to this file, you should do this at the end, or else any stored setting will get corrupted when switching ESPEasy builds